irohad/ametsuchi/impl/postgres_ordering_service_persistent_state.cpp

```diff
@@ -123,9 +128,11 @@ namespace iroha {
       return height;
     }
 
-    void PostgresOrderingServicePersistentState::reset() {
-      dropStorgage();
-      initStorage();
+    bool PostgresOrderingServicePersistentState::reset() {
+      bool result = true;
+      result &= dropStorgage();
+      result &= initStorage();
+      return result;
```

Comment:
Better to rewrite into one expression:
```
return dropStorage() & initStorage();
```
`& ` operator is a not lazy version of `and` operator.